pyccel/ast/builtin_methods/list_methods.py

```diff
@@ -14,7 +14,7 @@
 from pyccel.ast.internals import PyccelInternalFunction
 
 
-__all__ = ('ListAppend',)
+__all__ = ('ListAppend','ListInsert',)
```

Comment:
`__all__` is usually written with one object per line. This ensures it stays readable as it gets longer:
```suggestion
__all__ = ('ListAppend',
           'ListInsert',)
```

Reply:
updated in 9ed608928a7697654d55948ee1bbe70304500d9e